sdk/src/integrationTest/java/AccountBalanceIntegrationTest.java

```diff
@@ -134,6 +135,7 @@ void canFetchTokenBalancesForClientOperator() {
                 .execute(testEnv.client);
 
             assertEquals(balance.token.get(tokenId), 10000);
+            assertEquals(balance.decimal.get(balance.token.get(tokenId)), 50);
```

Comment:
@danielakhterov 

JS has `balance.tokens`. We should deprecate `.token` here and have `.tokens` and `.tokenDecimals`.

Reply:
Yea, that makes sense. Should double check that Go has `.Tokens` and not `.Token` as well.